packages/ember-intl/addon/services/intl.js

```diff
@@ -89,6 +89,15 @@ const IntlService = Service.extend(Evented, {
 
     return this.formatMessage(translation, ...args);
   },
+  
+  
+  tH(key, ...args) {
```

Comment:
I'm in agreement that this is needed, I just need to consider the API.  The method name is bothering me more than it should :)

We could possibly make it an option of the `t` method:
`this.t('foo.bar', { html: true })`